examples/struct_with_flatten.rs

```diff
@@ -8,6 +8,8 @@
 
 // Note: currently there is no automatic generation of "interactive clap::From Cli"
 
+#![allow(clippy::unit_arg, dead_code, unreachable_patterns, unused_variables)]
```

Comment:
it might've been bettern to `#[allow(...)]` each individual item/statement/expression , not the whole module